scripts/lsstinstall

```diff
@@ -348,6 +348,17 @@ else
     echo "$EUPS_PKGROOT" > "$EUPS_PATH/pkgroot" || fail "Unable to write pkgroot file"
 fi
 
+# Copy postinstall
+callback_folder="$EUPS_PATH/site"
```

Comment:
It looks like EUPS_PATH is not set during dry run mode so this could lead to a confusing error message. I was expecting the dry run test to come first in an outer if statement.